### PR TITLE
Correctly assign workflow_task.id in add-task

### PR DESCRIPTION
**Pull request by tcompa**

### CODE REVIEW

File: fractal/client/cmd/_workflow.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ async def workflow_add_task(
     **kwargs,
 ) -> RichJsonInterface:
     workflow_task = WorkflowTaskCreate(task_id=task_id, order=order)
+    workflow_task.workflow_id = id
```
Comment (jacopo-exact) [inline fractal/client/cmd/_workflow.py:76]: it doesn't harm, but it gets automatically added by the endpoint.

File: fractal/common/schemas/workflow.py [hunk 2]
```diff
@@ -26,6 +26,7 @@ class _WorkflowTaskBase(SQLModel):
 
 class WorkflowTaskCreate(_WorkflowTaskBase):
     task_id: int
+    # FIXME: should there be also workflow_id here?
```
Comment (jacopo-exact) [inline fractal/common/schemas/workflow.py:29]: it's inherited from above

**Review by jacopo-exact — COMMENTED** [review]
not fundamental but does no harm

**Comment by tcompa** [comment]
Issue #311 is not relevant, see https://github.com/fractal-analytics-platform/fractal-server/issues/225.
Closing.

**Pull request closed without merging**